src/main/scala/org/broadinstitute/dsde/firecloud/service/UserService.scala

```diff
@@ -107,12 +107,12 @@ trait UserService extends HttpService with PerRequestCreator with FireCloudReque
                         respondWithErrorReport(InternalServerError, InternalServerError.defaultMessage, requestContext)
                     }
                   case x =>
-                    // we got an unexpected error code from rawls; pass it on
-                    respondWithErrorReport(InternalServerError, "Unexpected response validating registration: " + x.toString, requestContext)
+                    // if we get any other error from rawls, pass that error on
+                    respondWithErrorReport(x.intValue, "Unexpected response validating registration: " + x.toString, requestContext)
```

Comment:
this means that if rawls returns a 503, we pass on the 503, instead of translating it to a 500.